src/drawing/DrawingPath.js

```diff
@@ -27,20 +27,31 @@ class DrawingPath {
      * @return {DrawingPath} A DrawingPath instance
      */
     constructor(drawingPathData: ?DrawingPath) {
-        if (drawingPathData) {
-            // $FlowFixMe
-            this.path = drawingPathData.path.map((num) => {
-                const x = +num.x;
-                const y = +num.y;
-
-                this.minX = Math.min(this.minX, x);
-                this.maxX = Math.max(this.maxX, x);
-                this.minY = Math.min(this.minY, y);
-                this.maxY = Math.max(this.maxY, y);
-
-                return createLocation(x, y);
-            });
+        this.initPath(drawingPathData);
+    }
+
+    /**
+     * Initialize a path from passed in data
+     * @param {DrawingPath} drawingPathData - The drawingPath object data to be instantiated into an object
+     * @return {void}
+     */
+    initPath(drawingPathData: ?DrawingPath) {
+        if (!drawingPathData) {
+            return;
         }
+
+        // $FlowFixMe
+        this.path = drawingPathData.path.map((num: Coordinates) => {
+            const x = +num.x;
+            const y = +num.y;
+
+            this.minX = Math.min(this.minX, x);
+            this.maxX = Math.max(this.maxX, x);
+            this.minY = Math.min(this.minY, y);
+            this.maxY = Math.max(this.maxY, y);
```

Comment:
Can you pull these four statements out into a function? Maybe something like `updatePathBounds()` or something like that?